components/MDXComponents.tsx

```diff
@@ -1,23 +1,24 @@
 /* eslint-disable import/no-cycle */
 /* eslint-disable @typescript-eslint/no-explicit-any */
 import React from 'react';
+import { camelCase } from 'lodash';
```

Comment:
why not lodash-es?

Reply:
it does the same thing...using lodash-es type build is causing compiling issues @hardik1003